plugins/HHAnalyzer.cc

```diff
@@ -91,9 +91,12 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
             ele.isMu = false;
             ele.isEl = true;
             ele.id_L = allelectrons.ids[ielectron][m_electron_loose_wp_name];
+            ele.id_M = ele.id_L; // FIXME: for now (January 2016) there is no Medium WP for electrons
```

Comment:
Are you sure? From this TWiki it seems to exist : https://twiki.cern.ch/twiki/bin/view/CMS/CutBasedElectronIdentificationRun2#Electron_ID_Working_Points_WP_de